src/test/java/org/broadinstitute/hellbender/tools/picard/sam/ValidateSamFileIntegrationTest.java

```diff
@@ -20,22 +21,34 @@ public String getTestedClassName() {
         return ValidateSamFile.class.getSimpleName();
     }
 
-    @DataProvider(name="testingData")
+    @DataProvider(name="tooltestingData")
     public Object[][] testingData() {
         return new Object[][]{
-            {"valid.sam", "SUMMARY", true},
-            {"valid.sam", "VERBOSE", true},
-            {"invalid_coord_sort_order.sam", "SUMMARY", false},
+          {"valid.sam", "SUMMARY", null, true},
+          {"valid.sam", "VERBOSE", null, true},
+          {"valid.bam", "VERBOSE", null, true},
+          {"valid.bam", "VERBOSE", "valid.fasta", false}, // the NM tags in this sam file don't match the reference deltas
+          {"valid.cram", "VERBOSE", "valid.fasta", true}, // the NM tags in the CRAM file do match the reference (ty samtools)
+          {"invalid_coord_sort_order.sam", "SUMMARY", null, false},
+          {"invalid_coord_sort_order.sam", "SUMMARY", "invalid_coord_sort_order.fasta", false},
+          //{"invalid_coord_sort_order.cram", "SUMMARY", "invalid_coord_sort_order.fasta", false},  // requires htsjdk fix to SamFileValidator
```

Comment:
We need a ticket in gatk to reenable this (that ticket may refer to the relevent issue in htsjdk)